src/content/learn/tutorials/general/releases.md

```diff
@@ -68,32 +72,35 @@ These constraints can be validated using downstream tools, such as the [OSCAL CL
 Accompanying pre-releases for these tools may be created as appropriate.
 For more details see ["How do I test downstream tooling?"](#how-do-i-test-downstream-oscal-tooling).
 
+{{% callout warning %}}Starting with [release 1.1.0](https://github.com/usnistgov/OSCAL/releases/tag/v1.1.0), build artifacts (schemas, converters) are now distributed as part of a release and are no longer tracked within the OSCAL Git repository.{{% /callout %}}
+
 #### Validating XML OSCAL content
 
 {{<callout>}}We appreciate the community using [example content from NIST](https://github.com/usnistgov/oscal-content/blob/main/examples/) and others, but we strongly encourage the community to test validation against real-world, production-grade data.{{</callout>}}
-XML OSCAL content can be validated using many tools that support XML Schema (XSD) files.
-Schemas are located in the OSCAL repository at the path [`/xml/schema`](https://github.com/usnistgov/OSCAL/tree/main/xml/schema).
 
+To obtain XML Schemas (XSDs) from a release, first follow the build instructions located in [`/build/README.md`](https://github.com/usnistgov/OSCAL/tree/main/build) or download the assets directly from the [corresponding GitHub release](https://github.com/usnistgov/OSCAL/releases).
+
+XML OSCAL content can be validated using many tools that support XML Schema (XSD) files.
 As an example, a piece of OSCAL content can be validated against a schema using [`libxml2`](https://gitlab.gnome.org/GNOME/libxml2)'s [`xmllint`](https://gnome.pages.gitlab.gnome.org/libxml2/xmllint.html):
 
 ```sh
 $ xmllint --noout \
-  --schema OSCAL/xml/schema/oscal_complete_schema.xsd \
+  --schema OSCAL/build/generated/oscal_complete_schema.xsd \
   example_oscal_content.xml
 ```
 
 #### Validating JSON and YAML OSCAL content
 
 {{<callout>}}We appreciate the community using [example content from NIST](https://github.com/usnistgov/oscal-content/blob/main/examples/) and others, but we strongly encourage the community to test validation against real-world, production-grade data.{{</callout>}}
 
-JSON and YAML OSCAL content can be validated using many tools that support [JSON Schema](http://json-schema.org/implementations.html).
-Schemas are located in the OSCAL repository at the path [`/json/schema/`](https://github.com/usnistgov/OSCAL/tree/main/json/schema).
+To obtain JSON schemas from a release, first follow the build instructions located in [`/build/README.md`](https://github.com/usnistgov/OSCAL/tree/main/build) or download the assets directly from the [corresponding GitHub release](https://github.com/usnistgov/OSCAL/releases).
 
+JSON and YAML OSCAL content can be validated using many tools that support [JSON Schema](http://json-schema.org/implementations.html).
```

Comment:
Minor, non-blocking: ditto, we typically same OSCAL DATAFORMAT content, not the other way around.

```suggestion
OSCAL JSON and YAML content can be validated using many tools that support [JSON Schema](http://json-schema.org/implementations.html).
```